OpenTelemetry.sln

```diff
@@ -186,6 +186,20 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "logs", "logs", "{3862190B-E
 		docs\logs\logging-correlation.md = docs\logs\logging-correlation.md
 	EndProjectSection
 EndProject
+<<<<<<< HEAD
```

Comment:
This looks like some conflict resolve header?

Reply:
Oh nice catch! Apparently, Visual Studio was like `this is fine`.